chapter4.adoc

```diff
@@ -233,26 +233,37 @@ supervisor domain are ascertained as follows:
 is 2^12^; MTT_PTE_SIZE = 8 bytes (for RV32, MTT_PTE_SIZE = 4 bytes). The `mttp`
 register must be active, i.e., the effective privilege mode must not be M-mode.
 
-2. Let _mpte_ be the value of the `MTT` table entry at address _a_ + _pa.pn[i]_
+2. If _pa_ has any bits set above the maximum physical address width supported
```

Comment:
```suggestion
2. If _pa_ is beyond the physical address space addressable
```

Reply:
The _pa_ width of a hart in RV64 is 64-bits. I think the constraint should be on the maximum accessible physical address space of the platform.

Reply:
This also needs to handle the case where the maximum physical address width of the hart (or platform) is greater than 46, but the hart is in Smmtt46 mode.

Reply:
That was the intent of suggesting maximum accessible physical address space and not the implemented physical address width. That would cover that case of Smmtt mode being more restrictive but that should be called out explicitly.